### DISCUSSION TITLE
Photoview

### DISCUSSION THREAD

**Discussion by nacho-domatix** [tteck/Proxmox#644] [Ideas]
After trying Photoprism for a while I found two big issues that makes it unusable for me:

1. The whole app is made is developed to "make you suscribe", free version does not have basic features like maps.
2. I couldn't find any elegant and simple way to link (read only) the repository of the photos, which are in a different system with Nextcloud and I don't want to let the app write there or re-copy all of them

After a while trying all the options, I believe [Photoview](https://photoview.github.io/) is the best alternative.

**Comment by johndoe0815** [comment]
Agree. Photoprism is too heavy, letting it process my photos would literally take months. 
Saw a Reddit post that besides Synology photos there are not many simple and good photo galleries. Photoview seems to be one of those rare candidates, so Support would be great.
